src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -49,7 +68,13 @@ class ConfirmationPage extends React.Component {
         {verified && userSignedIn && <div>
           <p>We’ll send you emails updating you on the status of your application. You can also print this page for your records. You should receive your Veteran ID Card by mail in about 60 days.<br/>
             In the meantime, you can print a temporary digital Veteran ID Card.</p>
-          <VeteranIDCard/>
+          <div className="id-card-preview">
+            <VeteranIDCard
+              veteranFullName={veteranFullNameStr}
+              veteranBranchCode={serviceBranch}
+              caseId={this.props.caseId}
```

Comment:
assuming this will come in as a prop since it's no on the schema.

Reply:
Think my comment got lost. It'll be `response.caseId`.